### ISSUE TITLE
Mark the person in the twitter post

### ISSUE THREAD

**Issue by filipeximenes** [vintasoftware/django-knowledge-share#18]
Sometimes people respond to tweets but since no one is watching the main account it goes unanswered. It would be good if the person who made the post was marked in the tweet so they get notified and can respond.

**Comment by felipefarias** [comment]
Good idea! We only need to include the person's account in the post message, right?

**Comment by filipeximenes** [comment]
yep
